app/graphql/types/user_type.rb

```diff
@@ -6,11 +6,23 @@ class UserType < Types::BaseObject
     field :name, String
     field :zipcode, String
     field :email, String
-    
+
     field :created_at, GraphQL::Types::ISO8601DateTime, null: false
     field :updated_at, GraphQL::Types::ISO8601DateTime, null: false
     field :user_books, [Types::UserBookType]
     field :books, [Types::BookType]
 
+
+    field :all_owned_books, [Types::BookType]
+    field :all_bookmarked_books, [Types::BookType]
+
+    def all_owned_books
+      object.user_books.where(status: 0).map { |user_book| user_book.book}
+    end
```

Comment:
thank you so much for figuring out the rest of this query! I was seriously so stuck after status.